test/compliance/generate_test_stft_data.py

```diff
@@ -0,0 +1,80 @@
+import random
+
+# Path to the compute-spectrogram-feats executable.
+EXE_PATH = '/scratch/jamarshon/kaldi/src/featbin/compute-spectrogram-feats'
+
+# Path to the scp file. An example of its contents would be "my_id /scratch/jamarshon/audio/test/assets/kaldi_file.wav"
+# where the space separates an id from a wav file.
+SCP_PATH = 'scp:/scratch/jamarshon/downloads/a.scp'
+# The directory to which the stft features will be written to.
+OUTPUT_DIR = 'ark:/scratch/jamarshon/audio/test/assets/kaldi/'
+
+# The number of samples inside the input wave file read from `SCP_PATH`
+WAV_LEN = 20
+
+# How many output files should be generated.
+NUMBER_OF_OUTPUTS = 100
+
+WINDOWS = ['hamming', 'hanning', 'povey', 'rectangular', 'blackman']
+
+
+def generate_rand_boolean():
+    # Generates a random boolean ('true', 'false')
+    return 'true' if random.randint(0, 1) else 'false'
+
+
+def generate_rand_window_type():
+    # Generates a random window type
+    return WINDOWS[random.randint(0, len(WINDOWS) - 1)]
+
+
+def run():
+    for i in range(NUMBER_OF_OUTPUTS):
+        inputs = {
+            'blackman_coeff': '%.4f' % (random.random() * 5),
+            'dither': '0',
```

Comment:
All the tests were done with `dither=0.` and the `energy_floor` set to a random value.